marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/entitySearch/saveSavedQueryTest.sjs

```diff
@@ -38,45 +40,150 @@ var saveQuery = JSON.stringify({
   }
 });
 
-function invokeService(saveQuery) {
-  return fn.head(xdmp.invoke(
-      "/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs",
-      {
-        "saveQuery": saveQuery
-      }
-  ));
+function beforeEach() {
```

Comment:
I think this is an indicator that multiple test modules should be created. That's because marklogic-unit-test has built-in support for "do something before every test" - just make a setup.xqy module.